raiden/tests/integration/test_integration_pfs.py

```diff
@@ -45,10 +45,16 @@ def test_pfs_send_capacity_updates_on_deposit_and_withdraw(
     transport0 = app0.raiden.transport
 
     pfs_room_name = make_room_alias(transport0.chain_id, PATH_FINDING_BROADCASTING_ROOM)
-    pfs_room = transport0._broadcast_rooms.get(pfs_room_name)
-    # need to assert for mypy that pfs_room is not None
-    assert isinstance(pfs_room, Room)
-    pfs_room.send_text = MagicMock(spec=pfs_room.send_text)
+
+    # Mock send_text on the PFS room
+    pfs_rooms: List[Room] = []
+    for app in [app0, app1]:
+        transport = app.raiden.transport
+        pfs_room = transport._broadcast_rooms.get(pfs_room_name)
+        # need to assert for mypy that pfs_room0 is not None
+        assert isinstance(pfs_room, Room)
+        pfs_room.send_text = MagicMock(spec=pfs_room.send_text)
```

Comment:
why do you need a mock for this?

Reply:
How would you do this instead?

Reply:
Postponed to https://github.com/raiden-network/raiden/issues/5652